### PR TITLE
Complete delete function

### PR DESCRIPTION
**Pull request by melaniesigrid**
## Hey there! 

Thank you for reviewing my code. On this PR, I... 

- Created functions to add, delete, and edit items in local storage.
- Edited style to make my app easier on the eyes!

Your comments will be appreciated!

### CODE REVIEW

File: webpack-demo/src/addRemove.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  const TaskItem = {
+    description: textInputValue,
+    completed: false,
+    index: retrievedList.length + 1,
+  };
+
+  const addedItem = TaskItem;
+  retrievedList.push(addedItem);
+  saveDataLocally(retrievedList);
+};
```
Comment (rukundoeric) [inline webpack-demo/src/addRemove.js]: - [x] <b>REQUIRED:</b> `Add` Functionality is not working. Kindly address the issue and fix it.
Screen Shot 2022-01-15 at 04 22 22

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hi @melaniesigrid, 

_Great job so far 👏 💯 👍. However, there are few issues we need to address before we move on. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 

## Required Changes ♻️
  - [x] Your current file structure would cause many unexpected errors and confusion. I strongly recommend you move everything from `webpack-demo/` to the root directory.
  - [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
  - [x] Check the comments under the review.

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: webpack-demo/src/addRemove.js (hunk 1 above)
Comment (melaniesigrid) [reply webpack-demo/src/addRemove.js]: Thank you for the comment and suggestion. After moving my files from webpack-demo/ to the root directory, I tested it and my console throws no error when adding, removing or editing elements.

Comment (melaniesigrid) [reply webpack-demo/src/addRemove.js]: ** This change generated linter errors with lodash and webpack imports that I was not able to correct. After reverting, I myself am not getting any errors when running my code for add, remove or edit using npm start. (???)  Unfortunately, I needed to generate a new branch to revert the changes back to keep the directory.

**Pull request closed without merging**